src/node/socket.ts

```diff
@@ -76,7 +75,10 @@ export class SocketProxyProvider {
       this._proxyServer = this.findFreeSocketPath(this.proxyPipe)
         .then((pipe) => {
           this.proxyPipe = pipe
-          return Promise.all([fs.mkdir(tmpdir, { recursive: true }), fs.rmdir(this.proxyPipe, { recursive: true })])
+          return Promise.all([
+            fs.mkdir(paths.runtime, { recursive: true }),
```

Comment:
I should have had it like this before:

```suggestion
            fs.mkdir(path.dirname(this.proxyPath), { recursive: true }),
```